client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -362,18 +310,9 @@ export function getEnhancedTasks(
 						},
 					};
 					break;
-				case 'videopress_setup':
-					taskData = {
-						completed: true,
-						disabled: true,
-						title: translate( 'Set up your video site' ),
-					};
-					break;
 				case 'videopress_upload':
 					taskData = {
-						title: translate( 'Upload your first video' ),
 						actionUrl: launchpadUploadVideoLink,
```

Comment:
I don't think actionUrl on videopress_upload is used anywhere. It's a hold over of the early launchpad days when we were using this attribute. While we're cleaning up unused attributes, we may want to remove this.